### PR TITLE
Module3

### PR DESCRIPTION
**Pull request by SpaceForCreativity**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,28 +1,32 @@
...
+import CreateCourse from './courses/CreateCourse';
+import CoursesList from './courses/CoursesList';
+import CourseInfo from './courses/CourseInfo';
+import { Routes as InternalRoutes } from './constants';
+import useCourses from './common/hooks/data/useCourses';
+import useAuthors from './common/hooks/data/useAuthors';
 
 function App() {
+  // for init
+  useCourses();
```
Comment (NikitaSedyx) [inline src/App.jsx:15]: by this you force your app to make requests when doesn't need this data, not the best approach

File: src/auth/Registration/index.jsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import { Link } from 'react-router-dom';
+import Input from '../../common/components/Input';
+import Button from '../../common/components/Button';
+import useUser from '../../common/hooks/data/useUser';
+import useInput from '../../common/hooks/utils/useInput';
+import './index.css';
+
+function Registration() {
```
Comment (NikitaSedyx) [inline src/auth/Registration/index.jsx:8]: index usually is used for exports

Comment (SpaceForCreativity) [reply src/auth/Registration/index.jsx:8]: Does it mean that I should import it in the main index.js? Or smth other?

File: src/App.jsx (hunk 1 above)
Comment (SpaceForCreativity) [reply src/App.jsx:15]: resolved in the next pull request

**Pull request merged**